packages/ui/react-ui-editor/src/hooks/useTextEditor.ts

```diff
@@ -92,17 +91,27 @@ export const useTextEditor = (
       // NOTE: Don't set selection here in case it is invalid (and crashes the state); dispatch below.
       const state = EditorState.create({
         doc: initialValue,
-        selection: initialSelection,
         extensions: [
           id && documentId.of(id),
+          extensions,
           // NOTE: Doesn't catch errors in keymap functions.
           EditorView.exceptionSink.of((err) => {
             log.catch(err);
           }),
-          extensions,
-          EditorView.updateListener.of(() => {
+          // TODO(burdon): Figure out better way.
```

Comment:
figure out a better way to do what? set the initial selection?

Reply:
yes, this is the best pattern for run-once on startup.


```
          ViewPlugin.fromClass(
            class {
              constructor(view) {}
            },
          ),
```